app/views/home/display_surveys.html.erb

```diff
@@ -0,0 +1,47 @@
+<div class="container">
+  <div class="row d-flex justify-content-between">
+    <h1>Surveys</h1>
+    <a class="btn btn-success my-auto" href="#_">Add Survey</a>
+  </div>
+  <div class="row d-flex justify-content-start">
+    <p class="text-muted">Show Surveys: </p>
+    <div class="dropdown mx-2 mb-2">
+      <button class="btn btn-secondary dropdown-toggle" type="button" id="dropdownMenuButton" data-toggle="dropdown">
+        Select
+      </button>
+      <div class="dropdown-menu" aria-labelledby="dropdownMenuButton">
+        <a class="dropdown-item" href="/home/display_expired_surveys">Expired</a>
+        <a class="dropdown-item" href="/home/display_active_surveys">Active</a>
+        <a class="dropdown-item" href="/home/display_surveys">All</a>
+      </div>
+    </div>
+  </div>
+  <div class="row justify-content-center">
+    <table class="table table-striped">
+      <thead class="thead-dark">
+        <tr>
+          <th scope="col">#</th>
+          <th scope="col">Title</th>
+          <th scope="col">Description</th>
+          <th scope="col">Category</th>
+          <th scope="col">Created By</th>
+          <th scope="col">Created on</th>
+          <th scope="col">Expiry</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @surveys.each_with_index do |survey, index| %>
+          <tr>
+          <th scope="row"><%= index+1 %></th>
+          <td><%= survey.name %></td>
+          <td><%= survey.description %></td>
+          <td><%= survey.category %></td>
+          <td><%= survey.user.first_name + " " + survey.user.last_name %></td>
```

Comment:
make a method for full_name in user model

Reply:
done